[Bug]: Example run commands in documentation module use a non-existent Maven module path

**Describe the bug**

The `**Run:**` Javadoc commands in `agentscope-examples/documentation` use
`-pl agentscope-examples/documentation2`, but the module's directory and artifactId
are `documentation`. 14 example commands fail immediately, and 6 of them also
reference non-existent or misplaced main classes (e.g. `StateAutoSaveExample` points
to `documentation2.session.SessionAutoSaveExample`, which does not exist).

**To Reproduce**

```
mvn exec:java -pl agentscope-examples/documentation2 \
    -Dexec.mainClass=io.agentscope.examples.documentation2.mcp.McpStdioExample
```

**Error messages**

```
[ERROR] Could not find the selected project in the reactor: agentscope-examples/documentation2
```

**Expected behavior**

Copy-pasted run commands from the example Javadocs should work as-is.

**Environment**

- AgentScope-Java Version: 2.0.3-SNAPSHOT (main)
- Java Version: 17
- OS: macOS

A fix PR will follow shortly.

--- a/agentscope-examples/documentation/pom.xml
+++ b/agentscope-examples/documentation/pom.xml
@@ -177,4 +177,15 @@
             <optional>true</optional>
         </dependency>
     </dependencies>
+
+    <build>
+        <plugins>
+            <!-- Declared without executions so the spring-boot:run prefix resolves; the default lifecycle is unaffected -->
+            <plugin>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>${spring.boot.version}</version>
+            </plugin>
+        </plugins>
+    </build>
 </project>

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/channel/ChannelSendExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/channel/ChannelSendExample.java
@@ -38,7 +38,7 @@ import java.util.List;
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
  *   mvn exec:java -pl agentscope-examples/documentation \
- *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.GatewayBasicExample
+ *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.channel.ChannelSendExample
  * </pre>
  */
 public class ChannelSendExample {

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/channel/GatewayMultiAgentExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/channel/GatewayMultiAgentExample.java
@@ -32,7 +32,7 @@ import io.agentscope.harness.agent.gateway.channel.chatui.SendOptions;
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
  *   mvn exec:java -pl agentscope-examples/documentation \
- *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.GatewayMultiAgentExample
+ *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.channel.GatewayMultiAgentExample
  * </pre>
  */
 public class GatewayMultiAgentExample {

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/subagent/SubagentSendDirectlyExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/subagent/SubagentSendDirectlyExample.java
@@ -44,7 +44,7 @@ import java.util.concurrent.atomic.AtomicReference;
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
  *   mvn exec:java -pl agentscope-examples/documentation \
- *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.GatewayStreamingThreadExample
+ *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.subagent.SubagentSendDirectlyExample
  * </pre>
  */
 public class SubagentSendDirectlyExample {

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/subagent/SubagentStreamingExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/harness/subagent/SubagentStreamingExample.java
@@ -54,7 +54,7 @@ import io.agentscope.harness.agent.subagent.SubagentDeclaration;
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
  *   mvn exec:java -pl agentscope-examples/documentation \
- *       -Dexec.mainClass=io.agentscope.examples.documentation2.streaming.SubagentStreamingExample
+ *       -Dexec.mainClass=io.agentscope.examples.documentation2.harness.subagent.SubagentStreamingExample
  * </pre>
  */
 public class SubagentStreamingExample {

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/hitl/PermissionHITLExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/hitl/PermissionHITLExample.java
@@ -60,7 +60,7 @@ import java.util.Scanner;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.hitl.PermissionHITLExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/mcp/McpSseExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/mcp/McpSseExample.java
@@ -44,7 +44,7 @@ import java.io.InputStreamReader;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.mcp.McpSseExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/mcp/McpStdioExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/mcp/McpStdioExample.java
@@ -42,7 +42,7 @@ import java.io.InputStreamReader;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.mcp.McpStdioExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/mcp/McpStreamableHttpExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/mcp/McpStreamableHttpExample.java
@@ -43,7 +43,7 @@ import java.io.InputStreamReader;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.mcp.McpStreamableHttpExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/middleware/ModelCallMiddlewareExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/middleware/ModelCallMiddlewareExample.java
@@ -59,7 +59,7 @@ import reactor.core.publisher.Flux;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.middleware.ModelCallMiddlewareExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/middleware/SystemPromptMiddlewareExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/middleware/SystemPromptMiddlewareExample.java
@@ -47,7 +47,7 @@ import reactor.core.publisher.Mono;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.middleware.SystemPromptMiddlewareExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/model/ModelRegistryExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/model/ModelRegistryExample.java
@@ -41,7 +41,7 @@ import io.agentscope.core.model.ModelRegistry;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.model.ModelRegistryExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/skill/AgentSkillExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/skill/AgentSkillExample.java
@@ -52,7 +52,7 @@ public class AgentSkillExample {
      * Output directory where the agent may write new skill files during the demo.
      */
     private static final String OUTPUT_DIR =
-            "agentscope-examples/documentation2/target/skill-output";
+            "agentscope-examples/documentation/target/skill-output";
 
     /**
      * Runs the agent skill example.

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/skill/SkillWithToolGroupExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/skill/SkillWithToolGroupExample.java
@@ -60,7 +60,7 @@ import java.nio.file.Paths;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.skill.SkillWithToolGroupExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/state/StateAutoSaveExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/state/StateAutoSaveExample.java
@@ -50,8 +50,8 @@ import java.nio.file.Paths;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
- *       -Dexec.mainClass=io.agentscope.examples.documentation2.session.SessionAutoSaveExample
+ *   mvn exec:java -pl agentscope-examples/documentation \
+ *       -Dexec.mainClass=io.agentscope.examples.documentation2.state.StateAutoSaveExample
  * </pre>
  */
 public class StateAutoSaveExample {

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/streaming/StreamingConsoleExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/streaming/StreamingConsoleExample.java
@@ -56,7 +56,7 @@ import java.util.Scanner;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.streaming.StreamingConsoleExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/streaming/StreamingWebExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/streaming/StreamingWebExample.java
@@ -47,8 +47,8 @@ import reactor.core.scheduler.Schedulers;
  * <p>Usage:
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn spring-boot:run -pl agentscope-examples/documentation2 \
- *       -Dspring-boot.run.mainClass=io.agentscope.examples.documentation2.StreamingWebExample
+ *   mvn spring-boot:run -pl agentscope-examples/documentation \
+ *       -Dspring-boot.run.mainClass=io.agentscope.examples.documentation2.streaming.StreamingWebExample
  *
  *   curl -N "http://localhost:8080/chat?message=Hello"
  *   curl -N "http://localhost:8080/chat?message=What+is+AI?&amp;sessionId=my-session"

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/tool/PermissionContextExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/tool/PermissionContextExample.java
@@ -54,7 +54,7 @@ import io.agentscope.harness.agent.middleware.AgentTraceMiddleware;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.tool.PermissionContextExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/tool/ToolBaseExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/tool/ToolBaseExample.java
@@ -56,7 +56,7 @@ import reactor.core.publisher.Mono;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.tool.ToolBaseExample
  * </pre>
  */

--- a/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/tool/ToolExecutionContextExample.java
+++ b/agentscope-examples/documentation/src/main/java/io/agentscope/examples/documentation2/tool/ToolExecutionContextExample.java
@@ -49,7 +49,7 @@ import java.util.List;
  * <p><b>Run:</b>
  * <pre>
  *   export DASHSCOPE_API_KEY=your_key
- *   mvn exec:java -pl agentscope-examples/documentation2 \
+ *   mvn exec:java -pl agentscope-examples/documentation \
  *       -Dexec.mainClass=io.agentscope.examples.documentation2.tool.ToolExecutionContextExample
  * </pre>
  */

